syntax = "proto3";


message ChatData {
    int32 team_index = 1;
    int32 room_index = 2;
    string chat = 3;
}

message returnData{
	string data = 1;
}

service tossChat {
    rpc tossChatData(ChatData) returns (returnData) {}
}